### ISSUE TITLE
Max display range of forms from dashboard showing 20

### ISSUE THREAD

**Issue by fergie-nz** [openmsupply/conforma-server#1029]
Possibly pagination issue?

![photo_2023-03-23_12-21-19](https://user-images.githubusercontent.com/90807420/227060366-da62f1cc-0d7f-4b06-8906-fcd4060a259c.jpg)

**Comment by CarlosNZ** [comment]
> Possibly pagination issue?

Yes, looks like it. Though I'd be interested to know why we're querying a page of actual records here rather than just the `totalCount` value. Seems potentially quite wasteful.

**Comment by jagoje** [comment]
@CarlosNZ @fergie-nz I can confirm that it only shows 20 in total, even when the links are more delineated. 

e.g. 16 awaiting assignment
4 under review 

etc.

**Comment by CarlosNZ** [comment]
> Though I'd be interested to know why we're querying a page of actual records here rather than just the `totalCount` value. Seems potentially quite wasteful.

I remember why now. Because when the list performance was really slow we changed this to just make a single query to the application list and do the filter separation in the front-end, rather than making a separate query per filter, which made it take a really long time.

We could possibly change this back now, but would need to test it out. Then we'd just need a single "get count" query for each filter and include the filter in the GraphQL query, and do away with front-end separation.
